test/deploy/k8s/cortx-pod-data.yml

```diff
@@ -1,39 +1,55 @@
-apiVersion: v1
-kind: Pod
+apiVersion: apps/v1
```

Comment:
Add these changes to a new file named cortx-deployment.yml

Reply:
Also change deploy.sh to call the new file.

Reply:
Also change deploy.sh to call the new file.

Reply:
done